### ISSUE TITLE
Using keyboard navigation to open a Application Launcher with Tooltip'd items fails

### ISSUE THREAD

**Issue by jeff-phillips-18** [patternfly/patternfly-react#2084]
Console error:

> typeError: Cannot read property 'focus' of undefined
>     at ApplicationLauncherMenu.componentDidMount
> 

Documentation page shows:
![image](https://user-images.githubusercontent.com/11633780/58343408-e35bac80-7e20-11e9-97c5-f1b03e4bc586.png)

**Comment by tlabaj** [comment]
Closed by #2297
